api/api.go

```diff
@@ -146,3 +158,19 @@ func (m *ApiService) handleLatestMerkleProof(w http.ResponseWriter, req *http.Re
 	}
 	m.respondOK(w, httpOkProofs{depositAddress, mRoot, slot, mPoof, avBalance, unbanBalance})
 }
+
+func (m *ApiService) handleLatestCheckpoint(w http.ResponseWriter, req *http.Request) {
+	log.WithFields(logrus.Fields{}).Info("/latestCheckpoint")
```

Comment:
if we dont use any `logrus.Fields{}` fields, perhaps we can just do:

`log.Info("/latestCheckpoint")`

`WithFields` is useful for example we want to log multiple variables in the same line.

```
log.WithFields(log.Fields{
			"aaa":  bbb,
			"ccc":  ddd,
			"eee": fff,
		}).Info("Scheduled Duty")
```